packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/javascript_dialog_request.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'package:flutter/foundation.dart';
+
+/// Defines the parameters that support setOnJavaScriptAlertDialog.
+@immutable
+class JavaScriptAlertDialogRequest {
+  /// Creates a [JavaScriptAlertDialogRequest].
+  const JavaScriptAlertDialogRequest({
+    required this.message,
+    required this.url,
+  });
+
+  /// Message to be displayed in the window.
+  final String message;
+
+  /// The url of the page requesting the dialog.
```

Comment:
Nit: URL

(In comments it should be capitalized normally, rather than as code.)